site/docs/configuration.md

```diff
@@ -59,6 +59,7 @@ Iceberg tables support table properties to configure table behavior, like the de
 | commit.manifest.target-size-bytes  | 8388608 (8 MB)   | Target size when merging manifest files                       |
 | commit.manifest.min-count-to-merge | 100              | Minimum number of manifests to accumulate before merging      |
 | commit.manifest-merge.enabled      | true             | Controls whether to automatically merge manifests on writes   |
+| commit.snapshot-id-inheritance.enabled | false        | Enables committing snapshots without explicit snapshot IDs    |
```

Comment:
Can we start a separate section for compatibility flags that explains in more detail? I'd like to highlight that this breaks forward-compatibility and add the minimum supported version. (Doesn't need to happen in this commit)

Reply:
Done. Let's add more details before the release.